AM_HW9_Notebook.ipynb

```diff
@@ -0,0 +1,461 @@
+{
```

Comment:
<p>Since you are using "z-accelaration" you calculating the actual straight line.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ubsuny/CP1_Calculus/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>